common/db_test.go

```diff
@@ -233,13 +233,16 @@ func TestSelectUpdateOrchs_AddingUpdatingRow_NoError(t *testing.T) {
 	assert.Equal(orchs[0].ServiceURI, orch.ServiceURI)
 
 	// updating row with same orchAddress
-	orchUpdate := NewDBOrch("127.0.0.1:8937", orchAddress)
+	orchUpdate := NewDBOrch("127.0.0.1:8937", orchAddress, 1000, 5, 10)
```

Comment:
Can we add tests for the different cases in `buildUpsertOrchsQuery()` i.e. for an existing row, update a column only if the field that corresponds to the column in the `DBOrch` struct does not equal the zero value?

Reply:
fixed in e8589ee4